readme-vars.yml

```diff
@@ -35,6 +35,8 @@ param_env_vars:
   - { env_var: "MONGO_HOST", env_value: "unifi-db", desc: "Mongodb Hostname. Only evaluated on first run." }
   - { env_var: "MONGO_PORT", env_value: "27017", desc: "Mongodb Port. Only evaluated on first run." }
   - { env_var: "MONGO_DBNAME", env_value: "unifi", desc: "Mongodb Database Name (stats DB is automatically suffixed with `_stat`). Only evaluated on first run." }
+  - { env_var: "MONGO_TLS", env_value: "true", desc: "Mongodb enable TLS. Only evaluated on first run." }
+  - { env_var: "MONGO_AUTHSOURCE", env_value: "admin", desc: "Mongodb authSource. For Atlas set to `admin`.Defaults to `MONGO_DBNAME`.Only evaluated on first run." }
```

Comment:
Set the `env_value` for both options to be the common defaults, in this case both blank, otherwise people copy/pasting without reading will end up with broken deployments.

Also, please move `MONGO_TLS` adn `MONGO_AUTHSOURCE` under `opt_param_env_vars`

Reply:
Done.